Makefile.js

```diff
@@ -648,8 +669,20 @@ target.gensite = function(prereleaseVersion) {
             let text = cat(filename),
                 title;
 
-            // 5. Prepend page title and layout variables at the top of rules
+            // 6. Prepend page title and layout variables at the top of rules
             if (path.dirname(filename).indexOf("rules") >= 0) {
+
+                // Find out if the rule requires a special docs portion (e.g. if it is recommended and/or fixable)
+                const isRecommended = recommended.has(ruleName);
```

Comment:
Gah, I just noticed that you actually requested adding `Set`s. Sorry I didn't look at this earlier:-( I still think adding an extra loop and two new data structures that we are not going to use for anything else later is an overkill, but I guess I can live with this, if everyone else thinks that it's fine.